stellar-contract-env-host/src/test.rs

```diff
@@ -89,3 +97,49 @@ fn vec_as_seen_by_host() -> Result<(), ()> {
     assert_eq!(val0, val1);
     Ok(())
 }
+
+#[fixture]
+fn empty_vec_obj() -> ScObject {
+    let scvec: ScVec = vec![].try_into().unwrap();
+    ScObject::Vec(scvec)
+}
+
+#[fixture]
+fn vec_obj() -> ScObject {
+    let scvec: ScVec = vec![ScVal::U32(1), ScVal::U32(2), ScVal::U32(3)]
+        .try_into()
+        .unwrap();
+    ScObject::Vec(scvec)
+}
```

Comment:
Fwiw I think the tests were easier to read before fixtures were added. Tests that are largely self contained are easier to read and maintain. DRY doesn't have the same benefits in tests.

Reply:
Hmm to me it makes writing tests easier with fewer lines of code. Since we will have many of these data structure tests operating on the same initial setup, I thought it might be worth.  But I definitely see your point. If you feel strongly about it I'm happy to revert it back.

Reply:
I would opt for test simplicity and test isolation at this point.

Reply:
Reverted it back.